includes/Admin/AnyCommentGenericSettings.php

```diff
@@ -524,33 +524,33 @@ public function init_settings () {
 		$form->add_section(
 			$this->section_builder()
 			     ->set_id( 'elements' )
-			     ->set_title( __( 'Elements', "anycomment" ) )
+			     ->set_title( __( 'Common Elements', "anycomment" ) )
 			     ->set_fields( [
 				     $this->field_builder()
 				          ->select()
 				          ->set_id( self::OPTION_FORM_TYPE )
-				          ->set_title( __( 'Comment form', "anycomment" ) )
+				          ->set_title( __( 'Who can comment?', "anycomment" ) )
```

Comment:
We have titles capitalized first letter, maybe we could make it the same way here, so that we won't need to fix it for many languages later

Reply:
```suggestion
				          ->set_title( __( 'Who Can Comment?', "anycomment" ) )
```